lib/_processQueue.ts

```diff
@@ -14,18 +16,29 @@ export function processQueue(globalObject) {
     const queue = globalObject[pointer].queue || [];
 
     // Loop queue and execute functions in the queue
-    queue.forEach((fn: string[]) => {
-      const functionName = fn[0];
-      const functionArguments = fn[1];
+    queue.forEach((fn: any[]) => {
+      const functionName: string = fn[0];
+      const functionArguments: any = fn[1];
+      const functionCallback: (err: any, res: any) => void = fn[2];
```

Comment:
we don't actually use `err` for now but it's a pattern enough that I implemented it.
Not very confident about this move. Let me know you thoughts.

Reply:
why not type this as 

```ts
queue.forEach(
  ([
    functionName: string,
    functionArguments: any,
    functionCallback: (err: any, res: any) => void
  ]) => {}
);
```

Reply:
We could even type the queue.

Reply:
@Haroenv If you mean
```ts
queue.forEach(
      ([functionName, functionArguments, functionCallback]: [
        string,
        any,
        Function
      ]) => {
      ...
      })
```
It's just that find the current code more legible. 
As françois says we can maybe just type the queue.
```ts
    type QueueItemCallback = (err: any, res: any) => void;
    type QueueItem = [string, any, QueueItemCallback?];
    const queue: QueueItem[] = [];
```

an then maybe just rely on the fact that the rest of the code will infer the typing.
```ts 
queue.forEach(([functionName, functionArguments, functionCallback])
```
WDYT?

Reply:
Yes, it makes more sense to me.

Reply:
fixed here: e9646a8